.teamcity/Gradle_Check/projects/StageProject.kt

```diff
@@ -39,8 +39,10 @@ class StageProject(model: CIBuildModel, stage: Stage, containsDeferredTests: Boo
     }
 
     stage.functionalTests.forEach { testCoverage ->
-        val isSplitIntoBuckets = testCoverage.testType != TestType.soak
-        if (isSplitIntoBuckets) {
```

Comment:
@wolfs Do you know why this is called `isSplitIntoBuckets`? I don't understand this, and I think this confuses people with the new bucket split thing, so I renamed this local variable.

Reply:
Since we used buckets (like numbers) for the projects a long time ago. Maybe we should call is something different. How about `useBuildTypeBuckets`? Since we called the splits `BuildTypeBucket`s?